hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/RealtimeFileStatus.java

```diff
@@ -34,51 +35,62 @@
  * in Path.
  */
 public class RealtimeFileStatus extends FileStatus {
```

Comment:
Do you want to rename this and other classes with `Realtime` word as well?

Reply:
Kept the naming as Realtime as an homage to previous state of things. Don't really want to rename pretty much all of the classes in Hive hierarchy (and essentially dock their history)

Reply:
realtime is a remnant from the design motivations for MOR snapshot queries. We might even revive it after the caching story